src/javascript/app_2/Stores/Modules/Contract/contract-store.js

```diff
@@ -41,52 +42,50 @@ export default class ContractStore extends BaseStore {
     @observable has_error         = false;
     @observable error_message     = '';
     @observable is_sell_requested = false;
+    @observable is_left_epoch_set = false;
```

Comment:
Could this be moved to the smart-chart-store? Since it seems related to the chart.

Reply:
It's related to SmartCharts but the variable is only relevant for when the contract is running.